_posts/2024-04-02-Multilingual-Search.md

```diff
@@ -0,0 +1,321 @@
+{"index": {"_index": "cohere_multilingual_english_catalog", "_id": 9}}
+{"gender": "Men", "masterCategory": "Accessories", "subCategory": "Watches", "articleType": "Watches", "baseColour": "Black", "season": "Winter", "year": "2016", "usage": "Casual", "productDisplayName": "Skagen Men Black Watch", "description": "Men Accessories Watches Watches Black Winter 2016 Casual Skagen Men Black Watch"}
+...
+```
+Once the data is ingested, the `description` field is converted into text embeddings and the embeddings are stored in `description_vector` using the ingestion pipeline and connector we created in earlier stages.
+
+### Performing multilingual search requests
+We can now search our catalog. Semantic search can be done in English, or all languages the [model supports](https://docs.cohere.com/docs/supported-languages).  
+
+#### Searching in English
+We can still run a semantic search in the English.
+```json
+GET /cohere_multilingual_english_catalog/_search
+{
+  "_source": [ "description", "subCategory"],
+  "size": 5,
+  "query": {
+    "neural": {
+      "description_vector": {
+        "query_text": "man birthday gift",
+        "model_id": "<MODEL_ID>",
+        "k": 5
+      }
+    }
+  }
+}
+```
+Our search query is `man birthday gift`, and notice that the results do not necessarily match the words `birthday` or `gift`:
+```json
+{
+  ...
+    "hits": [
+      {
+        ...
+        "_score": 0.5146248,
+        "_source": {
+          "subCategory": "Fragrance",
+          "description": "Men Personal Care Fragrance Deodorant Blue Spring 2017 Casual Rasasi Men Deserve Deo"
+        }
+      },
+      {
+        ...
+        "_score": 0.5110825,
+        "_source": {
+          "subCategory": "Jewellery",
+          "description": "Men Accessories Jewellery Ring Steel Summer 2013 Casual Revv Men Steel Ring"
+        }
+      },
+      {
+        ...
+        "_score": 0.5016889,
+        "_source": {
+          "subCategory": "Sandal",
+          "description": "Men Footwear Sandal Sandals Olive Summer 2012 Casual Bata Men Hummer Grey Sandals"
+        }
+      },
+      {
+        ...
+        "_score": 0.50156057,
+        "_source": {
+          "subCategory": "Fragrance",
+          "description": "Men Personal Care Fragrance Fragrance Gift Set Red Spring 2017 Casual Reebok Men Reeplay Perfume & Deodorant Set"
+        }
+      },
+      {
+        ...
+        "_score": 0.50010604,
+        "_source": {
+          "subCategory": "Sandal",
+          "description": "Men Footwear Sandal Sandals Brown Summer 2013 Casual Gas Men Aventura Sandal"
+        }
+      }
+    ]
+}
+```
+#### Searching in other languages
+Since the embedding was done using a multilingual embedding model, we can search in any of the languages supported by said model. 
```

Comment:
```suggestion
Because the embedding was performed using a multilingual embedding model, you can search in any of the languages supported by the model. 
```